main.tf

```diff
@@ -1 +1,62 @@
+resource "azuread_group" "aaddc_admins" {
+  display_name     = "AAD DC Administrators"
```

Comment:
Is "AAD DC Administrators" choosen by us? Or is this name required by EDS?
If we have control over it and there are no technical lmitations I would use the new name "EDS", right?

Reply:
Not chosen by us, the group needs to have this name. In the active directory itself every build in group also still uses AAD insted of Entra.